sdk/identity/identity-broker/README.md

```diff
@@ -0,0 +1,102 @@
+# Azure Identity Plugin for Brokered Authentication
+
+This package provides a plugin to the Azure Identity library for JavaScript ([`@azure/identity`](https://npmjs.com/package/@azure/identity)) that enables using an authentication broker such as [WAM](https://learn.microsoft.com/azure/active-directory/develop/scenario-desktop-acquire-token-wam).
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity-broker) | [Samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-broker/samples-dev)
+
+## Getting started
+
+```typescript
+import { nativeBrokerPlugin } from "@azure/identity-broker";
+import { useIdentityPlugin } from "@azure/identity";
+
+useIdentityPlugin(nativeBrokerPlugin);
+```
+
+### Prerequisites
+
+- An [Azure subscription](https://azure.microsoft.com/free/nodejs/).
+
+### Install the package
+
+This package is designed to be used with Azure Identity for JavaScript. Install both `@azure/identity` and this package using `npm`:
+
+```sh
+npm install --save @azure/identity
+npm install --save @azure/identity-broker
+```
+
+#### Supported Environments
+
+Azure Identity plugins for JavaScript support stable (even numbered) versions of Node.js starting from v18. While the plugins may run in other Node versions, no support is guaranteed. `@azure/identity-broker` **does not** support browser environments.
+
+## Key concepts
+
+If this is your first time using `@azure/identity` or Microsoft Entra ID, we recommend that you read [Using `@azure/identity` with Microsoft Entra ID](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/using-azure-identity.md) first. This document will give you a deeper understanding of the platform and how to configure your Azure account correctly.
+
+### Azure Identity Plugins
+
+As of `@azure/identity` version 2.0.0, the Identity client library for JavaScript includes a plugin API. This package (`@azure/identity-broker`) exports a plugin object that you must pass as an argument to the top-level `useIdentityPlugin` function from the `@azure/identity` package. Enable token cache persistence in your program as follows:
```

Comment:
This last sentence needs to be updated, as it describes a different plugin.

Reply:
done